### PR TITLE
Added commands ggpull, ggl, ggp & ggpush

### PR DESCRIPTION
**Pull request by DJWassink**
Thanks for these aliases, immediately missed them coming from zsh :)

Added the ggl & ggp commands which are currently missing, cheers!

### CODE REVIEW

File: src/aliases.ps1 [hunk 1]
```diff
@@ -369,3 +369,23 @@ function gwip {
 	git rm $(git ls-files --deleted) 2> $null
 	git commit --no-verify -m "--wip-- [skip ci]"
 }
+function ggpull {
```
Comment (gluons) [inline src/aliases.ps1]: `ggpull` is deprecated. Could you remove it?

File: src/aliases.ps1 [hunk 2]
```diff
@@ -369,3 +369,23 @@ function gwip {
...
+	$CurrentBranch = Get-Git-CurrentBranch
+
+	git pull origin $CurrentBranch
+}
+function ggl {
+	$CurrentBranch = Get-Git-CurrentBranch
+
+	git pull origin $CurrentBranch
+}
+function ggpush {
```
Comment (gluons) [inline src/aliases.ps1]: `ggpush` is deprecated. Could you remove it?

File: src/git-aliases.psm1 [hunk 3]
```diff
@@ -113,7 +113,11 @@ $FunctionsToExport = @(
 	'glum',
 	'gvt',
 	'gwch',
-	'gwip'
+	'gwip',
+	'ggpull',
```
Comment (gluons) [inline src/git-aliases.psm1]: `ggpull` is deprecated.

File: src/git-aliases.psm1 [hunk 4]
```diff
@@ -113,7 +113,11 @@ $FunctionsToExport = @(
...
+	'ggl',
+	'ggpush',
```
Comment (gluons) [inline src/git-aliases.psm1]: `ggpush` is deprecated.

**Comment by DJWassink** [comment]
Aah didn't know those were deprecated, sorry for the spam if any. Something went wrong while reverting the changes 😅

**Pull request merged**